Source/WebCore/platform/graphics/cairo/CairoUtilities.cpp

```diff
@@ -49,9 +49,13 @@
 namespace WebCore {
 
 static CairoUniquePtr<cairo_font_options_t> s_defaultCairoFontOptions;
+static CairoUniquePtr<cairo_font_options_t> s_defaultCairoFontOptionsForTesting;
```

Comment:
Why do we need this? Can't we just set s_defaultCairoFontOptions instead?

Reply:
Another variable is required because on GTK `s_defaultCairoFontOptions` will be overwritten by `GtkSettingsManagerProxy`:
https://github.com/WebKit/WebKit/blob/a9febb26c871be962432254ecb980a9a36ac7f13/Source/WebKit/WebProcess/gtk/GtkSettingsManagerProxy.cpp#L170

Reply:
That's confusing, in the case of GTK since we set specific settings via GtkSettings, we could just disable hinting that way. no?

Reply:
Yes, `GtkSettings` would work too. In fact, this is how it works now. I wanted to have one more explicit way to disable font hinting for testing which can be used on both platforms. I updated my PR. It doesn't require an additional variable only for testing. Moreover, `s_defaultCairoFontOptions` is always allocated only once. Font settings, such as hinting or antialiasing, are now applied separately from each other.